src/core/interpreter.cc

```diff
@@ -467,7 +469,7 @@ int RedisLogCommand(lua_State* lua) {
   int argc = lua_gettop(lua);
   if (argc < 2) {
     PushError(lua, "redis.log() requires two arguments or more.");
-    return RaiseError(lua);
+    RaiseError(lua);
```

Comment:
I think that keeping the `return` part here makes sense, like the lua API docs mention (this way readers can't be mislead to think that the flow will continue)
Unless you rename it like I suggested above, then I think it's ok as is